docs/papers/architecture-whitepaper.md

```diff
@@ -0,0 +1,1718 @@
+
+##### P2P Communication
+**Peer-to-peer (P2P) Communication** enables secure and direct communication between individual wallets for various purposes, such as transferring funds or exchanging information. This capability facilitates decentralized transactions and interactions without the need for intermediaries, providing users with greater control, privacy, and autonomy over their data. 
+
+```mermaid
+sequenceDiagram
+  participant w1 as Wallet 1
+  participant a as Wallet Addressing
+  participant w2 as Wallet 2
+  
+  w1->>+a: Register(w1_name, w1_protocol)
+  a-->>-w1: w1_address
+  w2->>+a: Register(w2_name, w1_protocol)
+  a-->>-w2: w2_address
+  
+  w1->>+a: Lookup(w2_name)
+  a-->>-w1: w2_address, w2_protocol
+  
+  w1->>+w2: Connect<w2_address, w2_protocol>(w1_name)
+  w2->>+a: Lookup(w1_name)
+  a-->>-w2: w1_address, w1_protocol
+  w2->>-w1: ACK<w1_address, w1_protocol>(w2_name)
+```
+
+###### Sub-capabilities
+- **Wallet Addressing** - Ability to uniquely identify a wallet to ensure that communications are routed correctly.
+- **Peer Discovery** - Enables wallets to identify and establish connections with other wallets.
+- **Secure Communication Protocols** - Ensure the confidentiality, integrity, and authenticity of P2P communications.
+
+###### Considerations
+- **Autonomy and Control** - Users maintain full control and ownership of their data and transactions, with the ability to transact directly with peers without third-party intervention.
+- **In-Person Processing** - P2P communication may be used in cases where data needs to be presented in person. Consider how contactless protocols may be utilized in these cases. In addition, consider how the wallet can be used without unlocking the underlying device or other items in the wallet.
+
+
+## Technology Enablers
+A **Technology Enabler** facilitates the implementation of the overall solution. Technology enablers can include software libraries, development frameworks, hardware components, cloud services, protocols, and standards. In this section, we break the enablers into cross-cutting enablers (those that support multiple solutions), identity enablers (those that support identity solutions), object enablers (those that support digital object solutions), and money enablers (those that support money solutions).
+
+### Cross Cutting Enablers
+- HTTPS/TLS
+- Public Key Cryptography
+- [Decentralized Key Recovery Protocols](https://derecalliance.org/)
+- Zero-Knowledge Proofs
+- Near Field Communication
+- Bluetooth Low-Energy
+- QR Codes
+- Magnetic Secure Transmission
+- Key Vaults
+- Secure Enclaves (TEE)
+- Hardware Security Modules
+- Biometrics
+- OAuth/OpenID Connect
+- FIDO2/Client-to-Authenticator Protocol (CTAP)
+- Trusted Service Managers (TSM)
+
+### Identity Enablers
+- DIDs
+- OIDC SIOP
+- Credential Formats
+    - W3C Verifiable Credentials
+    - AnonCreds
+    - ISO mDL
+    - ICAO DTC
+    - SD-JWT
+- P2P Protocols
```

Comment:
WebRTC? 

On our end we have working implementation we could submit. Requires a signaling service to discover each parties IP but then it's P2P

Reply:
Added. Thank you.